internal/manager/config_validation.go

```diff
@@ -105,6 +109,16 @@ func (c *Config) validateKongAdminAPI() error {
 	return nil
 }
 
+func (c *Config) validateFallbackConfiguration() error {
+	if !c.FeatureGates[featuregates.FallbackConfiguration] && c.UseLastValidConfigForFallback {
+		return fmt.Errorf(
+			"--use-last-valid-config-for-fallback can only be used with %s feature gate enabled",
```

Comment:
Since we have no mechanism as of today to check if the source of config flag is env var or CLI flag we might want to mention both here for completeness

Reply:
👍 9bf0bdf